src/style.css

```diff
@@ -1,3 +1,68 @@
 body {
   margin: 0;
 }
+
+.nav-link {
+  color: #000;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+  text-decoration: none;
+  position: relative;
+  line-height: 60px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.nav-list {
+  display: flex;
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item {
+  margin-right: 20px;
+  display: flex;
+  align-items: center;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.header__logo {
+  display: flex;
+  align-items: center;
+}
+
+.fixed-size-image {
+  width: 40px;
+  height: 40px;
+}
+
+.is-active {
+  color: #00ACDC;
+}
+
+.is-active::after {
+  content: '';
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  width: 100%;
+  height: 4px;
+  border-radius: 8px;
+  background: #00ACDC;
```

Comment:
Fix previous comment for creating a variable for the repeating color

Reply:
I don't understand what is required of me